.github/workflows/install.yml

```diff
@@ -0,0 +1,40 @@
+name: Install to Ubuntu
+on:
+  push:
+    branches:
+      - "*"
+    tags:
+      - "*"
+  schedule:
+    - cron: |
+        0 0 * * *
+jobs:
+  install:
+    name: Install to Ubuntu
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-18.04
+            package-type: apt
```

Comment:
Do we need this?
It seems that we can always use `apt`.

Reply:
Oh... This is needless.
I've fixed it.